discovery/README.md

```diff
@@ -10,8 +10,18 @@ Docker Swarm comes with multiple Discovery backends.
 
 ## Backends
 
+You use a hosted discovery service with Docker Swarm. The service
+maintains a list of IPs in your swam. There are several commercial
```

Comment:
Hm, re-reading this now, but don't think these are *commercial* services (I.e., the software is open-source, there probably are commercial hosted solutions *using* that software)